### PR TITLE
install ensembl 87 on travis

### PR DESCRIPTION
**Pull request by iskandr**
This change is [Reviewable](https://reviewable.io/reviews/hammerlab/varcode/219)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,16 +30,17 @@ install:
...
 script:
+  - ./lint.sh
   # older human Ensembl releases
   - pyensembl install --release 75 --species human
   - pyensembl install --release 77 --species human
   - pyensembl install --release 81 --species human
   # latest human Ensembl release
-  - pyensembl install --release 85 --species human
+  - pyensembl install --release 87 --species human
   # mouse tests written for mouse Ensembl #84 85
```
Comment (tavinathanson) [inline .travis.yml]: Fix this comment?

File: .travis.yml [hunk 2]
```diff
@@ -30,16 +30,17 @@ install:
...
-  - pyensembl install --release 85 --species mouse
+  - pyensembl install --release 87 --species mouse
   # now actually run the tests, generate a coverage report and run linter
-  - nosetests test --with-coverage --cover-package=varcode && ./lint.sh
+  - nosetests test --with-coverage --cover-package=varcode
```
Comment (tavinathanson) [inline .travis.yml]: What happened to the linter?

Comment (iskandr) [inline .travis.yml]: First line of the scripts

Comment (tavinathanson) [inline .travis.yml]: 👍

**Pull request merged**